protocol/rpcprovider/rpcprovider.go

```diff
@@ -142,7 +142,7 @@ type RPCProvider struct {
 	providerUniqueId          string
 	staticProvider            bool
 	staticSpecPath            string
-	providerLoadManager       *ProviderLoadManager
+	relayLoadLimit            uint64
```

Comment:
we don't need the rpcprovider to hold it. no use for it later on after creating all rpc provider servers.

Reply:
How can I use it in SetupEndpoint then? options all set up variables in rpcprovider to be used later